app/templates/introduction.html

```diff
@@ -54,12 +54,18 @@ <h2 class="u-fs-l">
             {% endif %}
 
             {%- if legal_basis -%}
-                <h2 class="u-fs-m u-mb-xs" data-qa="legal-response">{{ _("Your response is legally required") }}</h2>
+                <h3 class="u-fs-m" data-qa="legal-response">{{ _("Your response is legally required") }}</h3>
                 <p class="u-fs-r u-mb-m" data-qa="legal-basis">{{ legal_basis }}</p>
             {%- endif -%}
 
+            {% if survey_guide %}
+                <h3 class="u-fs-m" data-qa="survey-guide">{{ _("Guidance") }}</h3>
+                <p><a rel='noopener noreferrer' target='_blank' href={{ survey_guide }}>Guidance to help complete this survey</a></p>
```

Comment:
The static content should be wrapped in a `_()` i.e. `_("Guidance to help complete this survey")` because it is subject to being translated. There is also a process to update the pot file with the string to be translated. I think this is documented in the readme IIRC.

Reply:
Done